backend/app/Http/Controllers/CampaignController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace App\Http\Controllers;
+use App\Http\Controllers\Controller;
+
+use App\MyDefined\UseCase\Master\CreateCampaignUseCase;
+
+use App\MyDefined\ValueObject\CampaignNameValueObject;
+use App\MyDefined\ValueObject\ClientCodeValueObject;
+use App\MyDefined\ValueObject\DepartmentNameValueObject;
+use App\MyDefined\ValueObject\ManagerEmailValueObject;
+use App\MyDefined\ValueObject\ExecTimeValueObject;
+use App\MyDefined\ValueObject\OrderCategoryValueObject;
+use App\MyDefined\ValueObject\OrderNumberValueObject;
+
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+
+class CampaignController extends Controller
+{
+    public function createCampaign(
+        Request $request,
+        CreateCampaignUseCase $UseCase
+    )
+    {
+        $campaignName = CampaignNameValueObject::create($request->input('campaign_name'));
+        $orderDate = ExecTimeValueObject::create($request->input('order_date'));
+        $deadline = ExecTimeValueObject::create($request->input('deadline'));
+        $clientCode = ClientCodeValueObject::create($request->input('client_code'));
+        $department = DepartmentNameValueObject::create($request->input('department_name'));
+        $manager = ManagerEmailValueObject::create($request->input('manager_email'));
+        $orderCategory = OrderCategoryValueObject::create($request->input('order_category'));
+        $orderNumbers = [];
+        foreach($request->input('order_numbers') as $orderNumber){
+            array_push($orderNumbers, OrderNumberValueObject::create($orderNumber));
+        }
+
+        $result = $UseCase->execute($campaignName, $orderDate, $deadline, $clientCode, $department, 
+                                    $manager, $orderCategory, $orderNumbers);
+
```

Comment:
```suggestion
$result = $UseCase->execute(
    CampaignNameValueObject::create($request->input('campaign_name')),
    ExecTimeValueObject::create($request->input('order_date')),
    ExecTimeValueObject::create($request->input('deadline')),
    ClientCodeValueObject::create($request->input('client_code')),
    DepartmentNameValueObject::create($request->input('department_name')),
    ManagerEmailValueObject::create($request->input('manager_email')),
    OrderCategoryValueObject::create($request->input('order_category')),
    collect($request->input('order_numbers'))->map(function($orderNumber){
        return OrderNumberValueObject::create($orderNumber);
    })
);
```
配列操作よりもcollectionを使ってmap処理する方がスッキリ書ける（Laravelドキュメントのコレクションを参照）